python/mpt/src/mpt/trie_diff.cairo

```diff
@@ -981,6 +985,7 @@ func _left_is_extension_node{
     if (cast(right.value.leaf_node.value, felt) != 0) {
         // Remove the left node's key segment from the right leaf node
         let r_leaf = right.value.leaf_node;
+        check_leaf_node(path, r_leaf);
```

Comment:
why is this line considered not covered?

Reply:
this path might not be covered by the example we run the tests on